### PR TITLE
Design feedback for /projects

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Address feedback in https://github.com/ubuntudesign/vanilla-squad/issues/54

## QA

- Pull code
- Run `./run serve --watch`
- Check http://0.0.0.0:8002/projects and http://0.0.0.0:8002/projects/directory against the checklist above

## Issue / Card

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/54

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review

### Hero
- [ ] Update pictogram to: https://assets.ubuntu.com/v1/8d4630a6-pictogram-heart-mid-grey.svg

### Some projects we care about
- [ ] Remove margin-bottom from **p** tag as currently the divider line sits directly under body copy

**Comment by kwm14** [comment]
## Design review

### Some projects we care about
- [ ] Remove margin from card heading

**Comment by barrymcgee** [comment]
@kwm14 That margin comes from the vertical rhythm, do we need to make an issue upstream?

**Comment by kwm14** [comment]
@barrymcgee ok no worries. I have created an issue[ here](https://github.com/vanilla-framework/vanilla-framework/issues/1280) and we can schedule this work into our next iteration so it's fixed before releasing.

Will re-review this page with this bug in mind. Thanks

**Comment by kwm14** [comment]
@barrymcgee Looking good :+1: as discussed all .p-link--external links will be updated at global level.

File: static/sass/styles.scss [hunk 1]
```diff
@@ -119,3 +119,16 @@ time {
...
 
+
+// Add a table wrapper to allow x-scrolling
+.table__wrapper {
+  overflow-x: auto;
+  overflow-y: hidden;
+  margin: 2.5rem 0;
+}
+
+// p-card--foter needs margin-top
```
Comment (anthonydillon) [inline static/sass/styles.scss]: s/p-card--foter/p-card--footer

**Pull request merged**